tests/TarjetaTest.php

```diff
@@ -10,22 +10,44 @@ class TarjetaTest extends TestCase {
      * Comprueba que la tarjeta aumenta su saldo cuando se carga saldo válido.
      */
     public function testCargaSaldo() {
+        $val = [0, 10, 20, 30, 50, 100];
+        $valf = 510.15;
+        $valff = 962.59;
         $tarjeta = new Tarjeta;
+        $valorsaldo = 1;
+        $saldo = 0;
+        for ($valorsaldo; $valorsaldo < 1000; $valorsaldo++) {
+            if (in_array($valorsaldo, $val)) {
+                $this->assertTrue($tarjeta->recargar($valorsaldo));
+                $saldo += $valorsaldo;
+                $this->assertEquals($tarjeta->obtenerSaldo(), $saldo);
```

Comment:
Esta al reves, deberia ser `$this->assertEquals($saldo, $tarjeta->obtenerSaldo());`